From 66dceb57a23d12d319ff56bffb0af695704e0611 Mon Sep 17 00:00:00 2001
From: Tony Crisci <tony@dubstepdish.com>
Date: Wed, 25 Dec 2013 22:18:43 -0500
Subject: [PATCH] Simple fix for eof error

In start_child, if the command is null, just return.
---
 i3bar/src/child.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/i3bar/src/child.c b/i3bar/src/child.c
index 1bd0d25..527e353 100644
--- a/i3bar/src/child.c
+++ b/i3bar/src/child.c
@@ -456,6 +456,9 @@ void child_write_output(void) {
  *
  */
 void start_child(char *command) {
+    if (command == NULL)
+        return;
+
     /* Allocate a yajl parser which will be used to parse stdin. */
     memset(&callbacks, '\0', sizeof(yajl_callbacks));
     callbacks.yajl_map_key = stdin_map_key;
-- 
1.8.5.2

